src/components/Comments/DetailDomainScreenCommentItem.js

```diff
@@ -147,17 +147,13 @@ const styles = StyleSheet.create({
   container: ({isLast, style, level, isLastInParent}) => ({
     width: '100%',
     borderLeftWidth: isLastInParent ? 0 : 1,
-    borderLeftColor: isLast
-      ? level === 0
-        ? COLORS.lightgrey
-        : COLORS.transparent
-      : COLORS.lightgrey,
+    borderLeftColor: isLast ? (level === 0 ? COLORS.gray100 : COLORS.transparent) : COLORS.gray100,
     ...style
   }),
   username: {
     fontFamily: fonts.inter[700],
     fontSize: 12,
-    color: COLORS.blackgrey,
+    color: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The border color and username color have been updated. Make sure these changes are consistent across the application and improve readability for users.
```diff
- borderLeftColor: isLast ? (level === 0 ? COLORS.lightgrey : COLORS.transparent) : COLORS.lightgrey,
- color: COLORS.blackgrey,
+ borderLeftColor: isLast ? (level === 0 ? COLORS.gray100 : COLORS.transparent) : COLORS.gray100,
+ color: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->